flake.nix

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
@MarceloSpessoto , I think you added this commit by accident, right?

Reply:
yes, going to remove it